web/beacon-app/src/features/topics/components/AdvancedTopicPolicy.tsx

```diff
@@ -20,7 +20,7 @@ const AdvancedTopicPolicy = () => {
         <div className="flex space-x-1">
           <p>
             <Trans>
-              Coming soon! Customize and manage topic policies. The topic must be in the “Ready”
+              Coming soon! Customize and manage topic policies. The topic must be in the "Active"
```

Comment:
Changed the wording here to be consistent with what's returned from the BE and displayed in the topic state in the header component.